os/kernel/debug/secure_log.h

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <tinyara/config.h>
+
+/********************************************************************************
+ * Pre-processor Definitions
+ ********************************************************************************/
+
+/********************************************************************************
+ * Global Type Declarations
+ ********************************************************************************/
+
+/****************************************************************************
+ * Public Types
+ ****************************************************************************/
+
+/********************************************************************************
+ * Public Variables
+ ********************************************************************************/
+
+/****************************************************************************
+ * Inline Functions
+ ****************************************************************************/
+
+/********************************************************************************
+ * Public Function Prototypes
+ ********************************************************************************/
+#if defined(CONFIG_SE)
+/****************************************************************************
+ * Name: write_seclog
+ *
+ * Description:
+ *   This function write the value of secure log variable (which is saved in 
+ *   secure storage) in seclog_enable.
+ ****************************************************************************/
+int write_seclog(void);
+
+/****************************************************************************
+ * Name: read_seclog
+ *
+ * Description:
+ *   This function returns the seclog_enable.
+ ****************************************************************************/
+int read_seclog(void);
```

Comment:
It is local header.
To use in other module, It should be loacted public header.